bin/node/runtime/src/lib.rs

```diff
@@ -350,6 +351,7 @@ impl pallet_scheduler::Config for Runtime {
 	type ScheduleOrigin = EnsureRoot<AccountId>;
 	type MaxScheduledPerBlock = MaxScheduledPerBlock;
 	type WeightInfo = pallet_scheduler::weights::SubstrateWeight<Runtime>;
+	type OriginPrivilegeCmp = EqualPrivilegeOnly;
```

Comment:
There was some [improvement](https://github.com/paritytech/substrate/pull/10078) related to sheduled task cancelation. 
`EqualPrivilegeOnly` keeps old behavior after the improvement.